### ISSUE TITLE
[Feature Request:] Add code snippets

### ISSUE THREAD

**Issue by didacusAbella** [castwide/vscode-solargraph#140]
It would be great if you can add some yard snippets and ruby snippets to improve coding experience.
Thanks again for your amazing work!

**Comment by castwide** [comment]
Thanks!

I've been pondering snippet options. It doesn't seem like a good fit for the language server, as most editors (VS Code included) treat snippets as a discrete feature; but it might be a good fit for the extension. Also, it should probably be an opt-in feature to avoid conflicts with user-defined snippets and other extensions.

**Comment by didacusAbella** [comment]
@castwide you are right, and this feature request is to add this feature in VS-Code extension without touching the language server
